src/components/09-buttons/buttons.config.yml

```diff
@@ -1,8 +1,47 @@
+title: Buttons
 status: "ready"
+context:
+    label: "Primary"
 variants:
-    - name: "default"
-      label: "Buttons"
-    - name: "segmented"
-      label: "Segemented button"
-      notes: |
-            # Segmented Buttons
\ No newline at end of file
+    - name: "success"
+      context:
+          type: "success"
+          label: "Success"
+    - name: "danger"
+      context:
+          type: "danger"
+          label: "Danger"
+    - name: "plain"
+      context:
+          type: "plain"
+          label: "Plain"
+
+    - name: "secondary default"
+      context:
+          label: "Primary"
+          secondary: "secondary"
+    - name: "secondary success"
+      context:
+          type: "success"
+          label: "Success"
+          secondary: "secondary"
```

Comment:
Since a button is either secondary or not, what do you think about making it a boolean instead of a string? I see that you're using the value ("secondary") inside the template to apply the `--secondary` modifier class, but it might make more sense to check the truthiness of the `secondary` key and then apply the `--secondary` modifier based on that instead of interpolating the `{{ secondary }}` value in the template.

```yaml
secondary: true
```

Reply:
I briefly considered this as well and wasn't sure if the 'secondary' value could potentially change in the future. Using a boolean makes sense! I'll make that update.